react/src/components/Weather/APIValidation.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+const DisplayFeelTemp = ({ feelTemp }) => {
```

Comment:
tutaj to wszystko zwraca zawsze to samo można zrobić w jednym:
DisplayCoś = (cos) => !!cos ? <span>{cos}</span> : <>N/A</>;

Reply:
To było obgadywane chyba na innym PR
@ssmolenn

Reply:
@ssmolenn Pytanie powinno mieć odpowiedź, a nie zamknięcie wątku. Podobnie jak w:

https://github.com/paulidan/resposive-web/pull/14/files#r932071670